src/endpoints/onboarding_endpoints.py

```diff
@@ -92,13 +92,30 @@ def set_character_description(
         save_game_state(game_state, context)
 
     @post("/complete_onboarding")
-    def complete_onboarding(self, **kwargs) -> bool:
+    def complete_onboarding(self, **kwargs) -> bool:  # noqa: C901
         """Attempts to complete onboarding."""
         start = time.perf_counter()
         try:
             context = self.agent_service.build_default_context()
             game_state = get_game_state(context)
 
+            # These fields are required for the game to continue. If we're calling complete_onboarding and they're not
+            # set, we should fail early.
+            if not game_state.player.name:
```

Comment:
fail hard and early if these fields are empty -- they shouldn't be

Reply:
👍